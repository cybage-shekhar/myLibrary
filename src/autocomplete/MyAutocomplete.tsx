import React from "react";
export const MyAutocomplete = () => {
    return <input type='text' name="name"></input>
}