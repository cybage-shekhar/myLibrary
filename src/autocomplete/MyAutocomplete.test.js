import { shallow } from 'enzyme';
import React from 'react';
import { MyAutocomplete } from './MyAutocomplete';;
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
 
Enzyme.configure({ adapter: new Adapter() });

describe('Autocomplete', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<MyAutocomplete />);
    expect(wrapper).toMatchSnapshot();
  });
});