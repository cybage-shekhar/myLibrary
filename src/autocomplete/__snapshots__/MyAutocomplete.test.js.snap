// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Autocomplete renders correctly 1`] = `ShallowWrapper {}`;
